src/euphorie/client/model.py

```diff
@@ -147,7 +147,7 @@ class SurveyTreeItem(BaseObject):
         index=True,
     )
     type = schema.Column(
-        Enum(["risk", "module"]),
+        Enum(["risk", "choice", "module"]),
```

Comment:
I think that modifying an `Enum` requires an alembic upgrade, but I might be wrong.

Reply:
I used autogenerate to create the alembic upgrade and it didn't contain anything about the Enum. Also, it works fine as it is.

Reply:
Maybe you initialized the DB after the change in Python code was done?
Might I ask you to double check this?

This might help:

- https://stackoverflow.com/questions/14845203/altering-an-enum-field-using-alembic

If after that's not the case and an upgrade is not needed, you can resolve this conversation.

Reply:
I haven't initialized this DB in years.

The `type` column is just a varchar, there's nothing to upgrade there.

```
 type            | character varying(6)   |           | not null |
```